### PR TITLE
Add nitrate AOD look-up table and AOD calculation for UFS-Aerosols model

### PR DESCRIPTION
**Pull request by zhanglikate**
1)Add nitrate look-up table based on NASA optics_NI.v2_5.nc.
2)Implement the nitrate AOD calculation for UFS-Aerosols model, which is also included into the total AOD.

### CODE REVIEW

**Comment by WenMeng-NOAA** [comment]
@zhanglikate Can you sync your branch with the latest commit in UPP develop branch? The procedure can be found at 
https://github.com/NOAA-EMC/UPP/wiki/UPP-Code-Development#update-your-branch-at-your-fork-with-branch-develop-at-authoritative-repository

**Comment by zhanglikate** [comment]
> @zhanglikate Can you sync your branch with the latest commit in UPP develop branch? The procedure can be found at
> https://github.com/NOAA-EMC/UPP/wiki/UPP-Code-Development#update-your-branch-at-your-fork-with-branch-develop-at-authoritative-repository

@WenMeng-NOAA Finished. Thanks.

File: parm/post_avblflds.xml [hunk 1]
```diff
@@ -6312,14 +6330,15 @@
 
     <param>
        <post_avblfldidx>687</post_avblfldidx>
-       <shortname>NITRATE_AER_OPT_DEP_at550</shortname>
+       <shortname>NITRATE_AER_SCAT_OPT_DEP_at550</shortname>
```
Comment (WenMeng-NOAA) [inline parm/post_avblflds.xml:6333]: @zhanglikate Would you want to update the postcntrl_ufs_aerosol*xml files to reflect this short name change?

Comment (zhanglikate) [reply parm/post_avblflds.xml:6333]: NITRATE_AER_OPT_DEP_at550 has been moved to 615

**Comment by WenMeng-NOAA** [comment]
@zhanglikate I conducted the UPP standalone test with model history files provided by you. The results are at /scratch1/NCEPDEV/stmp2/Wen.Meng/post_gfs_2016071500. Can you verify my test? Thanks!

**Comment by zhanglikate** [comment]
> @zhanglikate I conducted the UPP standalone test with model history files provided by you. The results are at /scratch1/NCEPDEV/stmp2/Wen.Meng/post_gfs_2016071500. Can you verify my test? Thanks!

@WenMeng-NOAA  They are bit for bit identical. Thanks.

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were completed on WCOSS2 and Hera. No baseline update is needed for this PR.

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were completed on Orion.

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
@zhanglikate Can you sync your branch with upstream/develop one more time? Thanks!

**Comment by zhanglikate** [comment]
> @zhanglikate Can you sync your branch with upstream/develop one more time? Thanks!

@WenMeng-NOAA Finished. Thanks.

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were completed on Hera.

**Review by lipan-NOAA — COMMENTED** [review]
approve

**Review by lipan-NOAA — COMMENTED** [review]
approve

**Pull request merged**